testsuites/sparql/src/main/resources/testcases-sparql-1.0-w3c/data-r2/LICENSE

```diff
@@ -0,0 +1,21 @@
+All tests materials are licensed under the W3C Software Notice and License, as follows:
+____________________________________
+W3C� SOFTWARE NOTICE AND LICENSE
```

Comment:
This may just be an issue with Github, but there may be an undisplayable character here, likely UTF-8 versus latin1 or something like that but may want to look at it.